recipes/libstudxml/1.1.x/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.1)
+project(libstudxml LANGUAGES C CXX)
```

Comment:
Just the usual questions trying to understand - Why are we adding build scripts if the project has it's own?

I mean it's working for the older release why no stick with it?

Reply:
Because author has removed autotools & nmake build files to use build2 instead. I have no idea how build2 works, how to honor everything we have to honor (profile runtime, libcxx, custom cflags from profile, static or shared, install destination), nor if it does weird things which could defeat conan-center policies. Since a CMakeLists alternative is few lines long, it's far easier to write it.